v23.1/show-ranges.md

```diff
@@ -25,96 +29,567 @@ To use the `SHOW RANGES` statement, a user must either be a member of the [`admi
 
-### Show ranges for a table (primary index)
+### Show ranges for a database
+
+- [Show ranges for a database (without options)](#show-ranges-for-a-database-without-options)
+- [Show ranges for a database (with tables, keys, details)](#show-ranges-for-a-database-with-tables-keys-details)
+- [Show ranges for a database (with tables)](#show-ranges-for-a-database-with-tables)
+- [Show ranges for a database (with indexes)](#show-ranges-for-a-database-with-indexes)
+- [Show ranges for a database (with details)](#show-ranges-for-a-database-with-details)
+- [Show ranges for a database (with keys)](#show-ranges-for-a-database-with-keys)
+- [Show ranges for a database (with explain)](#show-ranges-for-a-database-with-explain)
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW DATABASES;
+~~~
+
+~~~
+  database_name | owner | primary_region | secondary_region | regions | survival_goal
+----------------+-------+----------------+------------------+---------+----------------
+  defaultdb     | root  | NULL           | NULL             | {}      | NULL
+  movr          | demo  | NULL           | NULL             | {}      | NULL
+  postgres      | root  | NULL           | NULL             | {}      | NULL
+  system        | node  | NULL           | NULL             | {}      | NULL
+(4 rows)
+~~~
+
+#### Show ranges for a database (without options)
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW RANGES FROM DATABASE movr;
+~~~
+
+~~~
+                                           start_key                                           |                                           end_key                                            | range_id | replicas |                                 replica_localities                                 | voting_replicas | non_voting_replicas | learner_replicas |    split_enforced_until
+-----------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------+----------+----------+------------------------------------------------------------------------------------+-----------------+---------------------+------------------+-----------------------------
+  /Table/106                                                                                   | /Table/106/1/"amsterdam"                                                                     |       70 | {1,6,8}  | {"region=us-east1,az=b","region=us-west1,az=c","region=europe-west1,az=c"}         | {1,6,8}         | {}                  | {}               | NULL
+  /Table/106/1/"amsterdam"                                                                     | /Table/106/1/"amsterdam"/"\xb333333@\x00\x80\x00\x00\x00\x00\x00\x00#"                       |       71 | {7,8,9}  | {"region=europe-west1,az=b","region=europe-west1,az=c","region=europe-west1,az=d"} | {9,7,8}         | {}                  | {}               | NULL
+  ...
```

Comment:
I think the ellipsis here breaks the markdown table. You might need to keep all the columns but fill each one with an ellipsis.

Same for the other tables below.